canonical-web-and-design/maas-ui
# feat(ui): remove KVM list action menu

## Done

- Removed KVM list action menu + checkboxes

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the KVM list and check that the checkboxes and action menu are gone and the "Add KVM" button is now positive
- Go to a KVM details page and check that the action menu is there

## Fixes

Fixes #2220
